index.js

```diff
@@ -1 +1,15 @@
-export * from './dist/auro-header';
+import { AuroHeader } from './src/auro-header';
+
+/**
+ * Register Custom Element.
+ * @param {Object} name - Name to use for custom element.
+ * @returns {void}
+ */
+ const registerComponent = (name = 'custom-header') => {
+  // alias definition
+  if (!customElements.get(name)) {
+    customElements.define(name, class extends AuroHeader {});
+  }
+}
+
+export { registerComponent }
```

Comment:
Please see this commit for an updated approach to this feature that addresses the issue with the automated API build. 

https://github.com/AlaskaAirlines/auro-badge/pull/45/commits/13100cbc41b16b334afceb3cb925d104658cd109